runtime/store.go

```diff
@@ -16,6 +16,14 @@ func NewKVStoreService(storeKey *storetypes.KVStoreKey) store.KVStoreService {
 	return &kvStoreService{key: storeKey}
 }
 
+func NewKVStoreKey(name string) *storetypes.KVStoreKey {
```

Comment:
Why do we need this? Shouldn't we use the mock kvstore service from core/testing in tests? Plus that removes no dependencies, it just makes it indirect.

Reply:
yeah i can move some of the core test stuff from Marko PR in to remove the use of storekey